source/includes/connect/sample-connect-app.java

```diff
@@ -0,0 +1,31 @@
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.ServerApi;
+import com.mongodb.ServerApiVersion;
+
+import org.bson.BsonDocument;
+import org.bson.BsonInt64;
+
+import com.mongodb.reactivestreams.client.MongoClient;
+import com.mongodb.reactivestreams.client.MongoClients;
+import com.mongodb.reactivestreams.client.MongoDatabase;
+import org.bson.conversions.Bson;
+import reactor.core.publisher.Mono;
+
+class ConnectionApp {
+    public static void main(String[] args) {
+        //start example code here
+        
+        //end example code here
+        {
+            Bson command = new BsonDocument("ping", new BsonInt64(1));
+            Mono.from(mongoClient.runCommand(command))
```

Comment:
`MongClient` does not have a `runCommand` method available in its API; `runCommand` is actually present in `MongoDatabase.`

Additionally, as we discussed and implemented in other PRs, we should split the creation of a publisher into a variable for clarity. For example, declare `Publisher<InsertResult> myPublisher = coll.insertOne(myDoc);` and then wrap it with `Mono.from(myPublisher)`.